build.sbt

```diff
@@ -350,3 +350,110 @@ lazy val `benchmarks-frontend` = project.in(file("benchmarks/frontend"))
     libraryDependencies ++= Dependencies.benchmarksFrontendDeps.value,
     Compile / scalaJSUseMainModuleInitializer := true,
   )
+
+lazy val `selenium-shared` = project.in(file("selenium/shared"))
```

Comment:
I'd much rather we setup the selenium project in a separate `build.sbt` file, no need to make this one more complex.